src/libs/kata-types/src/container.rs

```diff
@@ -7,11 +7,11 @@
 use std::fmt::{Display, Formatter};
 use std::str::FromStr;
 
-const CONTAINER: &str = "container";
-const SANDBOX: &str = "sandbox";
-const POD_CONTAINER: &str = "pod_container";
-const POD_SANDBOX: &str = "pod_sandbox";
-const POD_SANDBOX2: &str = "podsandbox";
+pub(crate) const CONTAINER: &str = "container";
+pub(crate) const SANDBOX: &str = "sandbox";
+pub(crate) const POD_CONTAINER: &str = "pod_container";
+pub(crate) const POD_SANDBOX: &str = "pod_sandbox";
+pub(crate) const POD_SANDBOX2: &str = "podsandbox";
```

Comment:
`2` is not ideal, how about `PODSANDBOX`, like other consts, they are all the upper case of the value.

Reply:
Thanks for the review, change `POD_SANDBOX2` to `PODSANDBOX`